triage/triaging.rst

```diff
@@ -62,44 +53,45 @@ For bugs, an issue needs to:
 These are things you can help with once you have experience developing for
 Python:
 
-* try reproducing the bug: For instance, if a bug is not clearly explained
-  enough for you to reproduce it then there is a good chance a core developer
+* Try reproducing the bug: for instance, if a bug is not explained clearly
+  enough for you to reproduce it, then there is a good chance a core developer
   won't be able to either.
-* see if the issue happens on a different Python version: It is always helpful
+* See if the issue happens on a different Python version: it is always helpful
   to know if a bug not only affects the in-development version of Python, but
   whether it also affects other versions in maintenance mode.
-* write a unit test: If the bug lacks a unit test that should end up in
+* Write a unit test: if the bug lacks a unit test that should end up in
   Python's test suite, having that written can be very helpful.
 
-This is all helpful as it allows triagers (i.e.,
-:ref:`people with the Developer role on the issue tracker <triagers>`) to
-properly classify an issue so it can be handled by the right core developers in
-a timely fashion.
+This is all helpful as it allows members of the :ref:`triage team <triage-team>`
+to properly classify an issue so it can be handled by the right core developers
+in a timely fashion.
 
-Reviewing Patches
------------------
+Reviewing Pull Requests
+-----------------------
 
-If an issue has a pull request attached that has not been reviewed, you can help
-by making sure the patch:
+If an issue has a linked pull request that has not been reviewed,
+you can help by making sure the pull request:
 
-* follows the style guides
-* applies cleanly to an up-to-date clone
 * is a good solution to the problem it is trying to solve
+* follows the style guides
 * includes proper tests
 * includes proper documentation changes
-* submitter is listed in ``Misc/ACKS``, either already or the patch adds them
+* includes a :ref:`NEWS entry <news-entry>` (if needed)
+* includes the author in ``Misc/ACKS``, either already or the patch adds them
```

Comment:
Use new role here?

Reply:
That PR is not merged yet -- we can add it later throughout the devguide.